packages/cli/tests/unit/fingerprint/fingerprintWatchCommand.spec.ts

```diff
@@ -93,6 +93,21 @@ describe(FingerprintWatchCommand, () => {
       expect(cmd.watcher).toBeUndefined();
      });
 
+    it('does not raise if it hits the limit of the number of open files', async () => {
+      cmd = new FingerprintWatchCommand(appMapDir);
+      await cmd.watcherErrorFunction(new Error("EMFILE: too many open files"));
+      expect(cmd.watcher).toBeUndefined();
+
+      cmd.watcher = new FSWatcher();
+      expect(cmd.watcher).not.toBeUndefined();
+      console.warn = jest.fn();
```

Comment:
This will replace the function globally and not restore it after the test. Use `stub(console, 'warn')` instead.